app/src/pages/inside/stepPage/modals/editDefectModals/makeDecisionModal/makeDecisionModal.jsx

```diff
@@ -129,11 +129,21 @@ const MakeDecision = ({ data }) => {
         },
       }));
     }
+    let comment = '';
```

Comment:
You can specify the default comment value as `comment = itemData.issue.comment || ''` and shorten next modifications.